docs/getting-started/ml-req.asciidoc

```diff
@@ -4,7 +4,6 @@
 To run and create {ml} jobs and rules, you need all of these:
 
 * The *https://www.elastic.co/subscriptions[appropriate license]*
-* {ml-cap} enabled on all master-eligible and coordinating {es} nodes (see
-{ref}/modules-node.html#ml-node[{ml-cap} node])
+* There must be at least one {ml} node in your cluster (see {ml-docs}/setup.html[{ml-cap} node])
```

Comment:
Hi @benironside @joepeeples - confused if we are linking to the top of the doc here? The title says `"Set up machine learning features"`, not the `"Machine learning node"` link text mentioned.

Reply:
Ah yes, we updated the link but forgot to update the link text. This should do it (looks like the title of that page has stayed the same over the previous versions).
```suggestion
* There must be at least one {ml} node in your cluster (see {ml-docs}/setup.html[Set up {ml} features])
```